doc/Language/operators.pod6

```diff
@@ -1822,6 +1822,28 @@ implement an appropriate infix C<eqv> operator:
     multi infix:<eqv>(A $l, A $r) { $l.a eqv $r.a }
     say A.new(a => 5) eqv A.new(a => 5);            # OUTPUT: «True␤»
 
+B<Trap:> If these equivalent C<A> objects are inside a container like
```

Comment:
Not sure about this, or the way it's formulated. If it's a Trap, there's a specific page for that. If you want to keep it here, it's better if you formulate it in a different way...

Reply:
Yes, let me try to remove that and make it sound more directly relevant to the documentation of eqv, instead of discussing a special case involving eqv:

------

Note that eqv does not work recursively on every kind of container type, e.g. Set:
```
my class A {
    has $.a;
}
say Set(A.new(a => 5)) eqv Set(A.new(a => 5));  # OUTPUT: «False␤»
```

Even though the contents of the two sets are `eqv`, the sets are not. The reason is that
`eqv` delegates the equality check to the `Set` object which relies on element-wise `===`
comparison. Turning the class `A` into a *value type* by giving it a `WHICH` method
produces the expected behavior: [...]

Reply:
OK, thanks.